### DISCUSSION TITLE
¿Cómo agregar una nueva conexión postgis al navegador QGIS con pyqgis?

### DISCUSSION THREAD

**Discussion by percyelbis** [qgispe/discusiones#16] [Preguntas&Respuestas] [answered]
Hola!!! Comunidad de QGIS Perú, quisiera saber si podrían apoyarme con la siguiente pregunta, gracias por su tiempo y sus amables respuestas.
**
![postgis](https://user-images.githubusercontent.com/30003438/126329475-2f83ed3b-e460-4fbf-b6ab-02b8ba161fdc.png)
**

**Comment by ambarja** [comment, marked as answer]
Hola @percyelbis  puedes usar el siguiente code 
```{python}
import psycopg2

conn = psycopg2.connect("dbname='postgis'host='localhost' user='postgis'")
cur = conn.cursor()
sql = "SELECT ST_AsGeoJSON(geomfield) from testtable;"
cur.execute(sql)
result = cur.fetchone()
print result

```
Aquí te adjunto algunas referencias :arrow_down: 
 - [click aquí](https://gis.stackexchange.com/questions/29721/how-to-connect-a-pyqgis-plugin-with-postgres)
